src/test/resources/features/Demo2.feature

```diff
@@ -2,7 +2,7 @@ Feature: Create a Card in Trello
 
```

Comment:
We should be to create specific feature file for each test, Demo2.feature is not good nor descriptive name